06_puppy/kasunfdo/mapstore.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+func (m MapStore) CreatePuppy(puppy *Puppy) bool {
+	if _, ok := m[puppy.ID]; !ok {
```

Comment:
"exists" might be a better variable name here?

Reply:
I agree. :)